x-pack/solutions/security/plugins/security_solution/public/common/components/filters_global/filters_global.test.tsx

```diff
@@ -8,6 +8,7 @@
 import type { ReactWrapper } from 'enzyme';
 import { mount, shallow } from 'enzyme';
 import React from 'react';
+import 'jest-styled-components';
```

Comment:
It's going to be difficult to trace the purpose/history of these imports later on. Can we add a comment so that's more clear?

```suggestion
// Necessary until components under test are migrated off of styled-components
import 'jest-styled-components';
```